src/body.js

```diff
@@ -144,6 +144,10 @@ Body.prototype = {
 	 * @return  Promise
 	 */
 	textConverted() {
+		if (!convertBody) {
```

Comment:
which means, we no longer has this API, we dropped it.

the users will have to rewrite their code from `res.textConverted()` to something like

```
var convertBody = require('fetch-charset-detection');

var buffer = res.buffer();
var length = res.headers.get('content-length');
var text = convertBody(buffer, length);
```